components/Article/index.tsx

```diff
@@ -0,0 +1,15 @@
+import style from './Article.module.css'
+
+export type ArticleProps = {
+  title: string
+  content: string
+}
+
+export default function Article({ title, content }: ArticleProps): JSX.Element {
+  return (
+    <main className={style.box}>
+      <h1 className={style.title}>{title}</h1>
```

Comment:
This is a good point. People always misuse headers. There should only be one `<h1>` per page. If there are multiple articles on a page, they should each begin with an `<h2>`. Subheadings under that should be `<h3>`. It is doubtful that we'll ever need to go below `<h3>`.

Too many devs use `hn` tags to control font size. That is an atrocity. The _n_ is used to indicate position in the _hierarchy_. Headings should be properly nested. This is not only good semantics (for machine learning, e.g.) but also good accessibility (to persons with varying abilities).

The question is, in an application with _user-generated_ content, how can one restrict the use of `hn` elements so that they are used correctly? Frankly, I have always thought that they were a mistake. There should be a single `<heading>` tag (the page heading should be called the `<header>`), and `<section>` tags. The page would be divided into sections and subsections with `<section>` tags and each would permit a `<header>` tag, which would automatically set the level of heading, subheading, etc. according to the nesting of the sections.

Actually, the `<header>` element is _not_ limited to one per page at the top of the page (ditto, `<footer>`). Each section could have a `<header>` tag and when HTML5 was first introduced there was some talk of _all_ the `hn` tags being `<h1>`, but nested in proper headers in sections (or articles or asides, which are the same thing -- sectioning elements).

We'll have to think about this. Maybe we can be the first CMS builders to get it right.